docs/core/diagnostics/managed-debuggers.md

```diff
@@ -14,14 +14,13 @@ Microsoft provides debuggers for managed code in **Visual Studio** and **Visual
 **Visual Studio** is an integrated development environment with the most comprehensive debugger available. Visual Studio is an excellent choice for developers working on Windows.
 
 - [Tutorial - Debugging a .NET Core application on Windows with Visual Studio](../tutorials/debugging-with-visual-studio.md)
+- [Debug ASP.NET Core apps in Visual Studio](/visualstudio/debugger/how-to-enable-debugging-for-aspnet-applications#debug-aspnet-core-apps)
 
-While Visual Studio is a Windows application, it can still be used to debug Linux and macOS apps remotely.
-
-- [Debugging a .NET Core application on Linux/OSX with Visual Studio](https://github.com/Microsoft/MIEngine/wiki/Offroad-Debugging-of-.NET-Core-on-Linux---OSX-from-Visual-Studio)
-
- Debugging ASP.NET Core apps require slightly different instructions.
+While Visual Studio is a Windows application, it can also be used to debug Linux apps running remotely, in WSL, or in docker containers:
```

Comment:
```suggestion
While Visual Studio is a Windows application, it can also be used to debug Linux apps running remotely, in WSL, or in Docker containers:
```